sale_blanket_order/models/blanket_orders.py

```diff
@@ -260,6 +262,21 @@ def onchange_partner_id(self):
             values["team_id"] = self.partner_id.team_id.id
         self.update(values)
 
+    @api.onchange("fiscal_position_id")
+    def _onchange_fiscal_position(self):
+        for order_line in self.line_ids:
+            if self.env.uid == SUPERUSER_ID:
```

Comment:
Strange that we're handling a case with `SUPERUSER_ID`, why it is needed?

Reply:
Based on [this chunk](https://github.com/versada/sale-workflow/blob/450fc3858d6b25b05402c385afd931d660e856a7/sale_blanket_order/models/blanket_orders.py#L646), which was introduced during migration 10.0 -> 11.0 [without any explanation](https://github.com/OCA/sale-workflow/pull/750) ¯\\_(ツ)_/¯

Reply:
This is nonsense created in OCA module. Would recommend not copy that. Better, copy the logic from the sale module (because blanked sale order is essentially a sale) https://github.com/odoo/odoo/blob/13.0/addons/sale/models/sale.py#L1162-L1167